### ISSUE TITLE
Can't `pip install python-mangopay`

### ISSUE THREAD

**Issue by chocopoche** [Mangopay/mangopay2-python-sdk#66]
I'm trying to upgrade to v3 because v2.x.x will be deprecated in 10 days.

When trying to install python-mangopay, setup.py fails because DESCRIPTION.rst does not exist.

Also it downloads version python-mangopay-1.0.0a0 which has been uploaded in 2015 on PyPI. I'm not sure about what should I do now.

```console
(venv) ➜  myproject pip install python-mangopay --no-cache-dir
Collecting python-mangopay
  Downloading python-mangopay-1.0.0a0.tar.gz (82kB)
    100% |████████████████████████████████| 92kB 922kB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-mnr9gqfg/python-mangopay/setup.py", line 7, in <module>
        with open(path.join(here, 'DESCRIPTION.rst'), encoding='utf-8') as f:
      File "/tmp/myproject/venv/lib64/python3.5/codecs.py", line 895, in open
        file = builtins.open(filename, mode, buffering)
    FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-build-mnr9gqfg/python-mangopay/DESCRIPTION.rst' 
```

**Comment by hobailey** [comment]
Hey there, v3 was yet published to Pypi, but we've just done that. The name of the install was also wrong so we've just updated the readme. Let us know if you're still have problems and thanks for flagging this up :-)

**Comment by chocopoche** [comment]
Well, pip install still fails when looking for DESCRIPTION.rst

**Comment by hobailey** [comment]
Hmm, very strange. I've made some changes and released v3.0.2 which should hopefully be OK.

**Comment by chocopoche** [comment]
Fixed on my side. Thank you!
